src/web/lib/ElementRenderer.tsx

```diff
@@ -286,6 +287,17 @@ export const ElementRenderer = ({
 					/>
 				</Figure>
 			);
+		case 'model.dotcomrendering.pageElements.FullReviewLink':
+			return (
+				<>
+					<FullReview
+						href={element.href}
+						title={element.title}
+						pillar={format.theme}
+					/>
+					<DividerBlockComponent size="full" />
```

Comment:
As per the divider comment above, you could maybe try using two `DividerBlockComponent`s here? Not sure if that works. I think we should avooid `hr`s if we can

And actually, this html should only return `FullReivew`, the Dividers should be inserted by the enhancer - there's examples of this being done in other enhancers